pkg/helpers/pki_test.go

```diff
@@ -235,3 +235,15 @@ func TestCreatePkiKeyCertPair(t *testing.T) {
 		t.Errorf("unexpected error thrown while executing CreatePkiKeyCertPair : %s", err.Error())
 	}
 }
+
+func TestSetSetPkiKeySize(t *testing.T) {
```

Comment:
there's a typo here, should be `TestSetSetPkiKeySize`

Reply:
good catch, will update it.